assets/i18n/ar.json

```diff
@@ -1743,7 +1745,9 @@
 	"I18N_TOPNAV_GET_INVOLVED": "شارك",
 	"I18N_TOPNAV_GET_STARTED": "ابدأ",
 	"I18N_TOPNAV_HOME": "الرئيسية",
-	"I18N_TOPNAV_IMPACT_REPORT": "تقرير بتأثير Oppia",
+	"I18N_TOPNAV_IMPACT_REPORT": "تقارير التأثير Oppia",
+    "I18N_TOPNAV_IMPACT_REPORT_2022": "2022",
```

Comment:
Please indent this line and the line below to match the other lines in this file. Ditto elsewhere.